File: scripts/scala/scala-playground/test/CalcTest.scala
import org.junit.Test
import junit.framework.Assert

class CalcTest {
	
	@Test
	def testSum1Plus1() = {
		val calc   = new Calc()
		val result = calc.sum(1, 1)
		Assert.assertEquals("1 + 1 deveria ser 2, erro na soma",2, result)		
	}
	
	@Test
	def testSum1minus1() = {
		val calc   = new Calc()
		val result = calc.sub(1, 1)
		Assert.assertEquals("1 - 1 deveria ser 0, erro na soma",0, result)		
	}
	
}

File: scripts/python/test-python/src/calc_test.py
import unittest
import Calculator as calc

class TestCalculator(unittest.TestCase):

    def test_shuffle(self):
        c      = calc.Calculator()
        result = c.sum(5, 3)
        self.assertEqual(result, 8,"A Soma 5 + 3 deveria ser = 8 ! Resultado: %s" % result )

if __name__ == '__main__':
    unittest.main()